src/sass/style.scss

```diff
@@ -207,3 +207,51 @@ section#subscribe {
     text-align: center;
   }
 }
+
+section#sponsors {
+  h1#sponsors {
+    font-family: "oxygenbold";
```

Comment:
Take a look how I use font classes in other places. I'm using them because `font-family` alone is not enough - you need to put `font-width` and so on.